internal/controllers/objectdeployments/adapter_test.go

```diff
@@ -98,7 +98,10 @@ func TestGenericObjectSet_GetObjects(t *testing.T) {
 			},
 				nil,
 			),
-			expectedObjectIDs: []string{"/ConfigMap/namespace1/cm1", "/ConfigMap/namespace2/cm2", "apps/Deployment/default/deployment1", "/Secret/default/secret1"},
+			expectedObjectIDs: []string{
+				"/ConfigMap/namespace1/cm1", "/ConfigMap/namespace2/cm2",
+				"apps/Deployment/default/deployment1", "/Secret/default/secret1",
+			},
```

Comment:
Why not 

```suggestion
			expectedObjectIDs: []string{
				"/ConfigMap/namespace1/cm1",
				"/ConfigMap/namespace2/cm2",
				"apps/Deployment/default/deployment1",
				"/Secret/default/secret1",
			},
```

?

To me that is way easier to read when glancing over and searching for "that one string"